src/flint/fmpq.jl

```diff
@@ -713,6 +713,8 @@ convert(::Type{fmpq}, a::Integer) = fmpq(a)
 
 convert(::Type{fmpq}, a::fmpz) = fmpq(a)
 
+Base.convert(::Type{fmpq}, x::Rational) = fmpq(x.num, x.den)
```

Comment:
Does convert need the Base qualifier? We don't seem to have it on the previous line.